com.woltlab.wcf/templates/article.tpl

```diff
@@ -121,76 +62,72 @@
 						{else}
 							{$articleContent->getImage()->caption}
 						{/if}
 					</figcaption>
 				{/if}
 			</figure>
 			<meta itemprop="url" content="{$articleContent->getImage()->getThumbnailLink('large')}">
-			<meta itemprop="width" content="{@$articleContent->getImage()->getThumbnailWidth('large')}">
-			<meta itemprop="height" content="{@$articleContent->getImage()->getThumbnailHeight('large')}">
+			<meta itemprop="width" content="{$articleContent->getImage()->getThumbnailWidth('large')}">
+			<meta itemprop="height" content="{$articleContent->getImage()->getThumbnailHeight('large')}">
 		</div>
 	{/if}
 	
 	{event name='beforeArticleContent'}
-	
-	<div class="section articleContent" {@$__wcf->getReactionHandler()->getDataAttributes('com.woltlab.wcf.likeableArticle', $article->articleID)}>
-		<div class="section htmlContent" itemprop="description articleBody">
-			{if MODULE_WCF_AD}
-				{@$__wcf->getAdHandler()->getAds('com.woltlab.wcf.article.inArticle')}
-			{/if}
-			
-			{@$articleContent->getFormattedContent()}
-			
-			{event name='htmlArticleContent'}
-		</div>
 
-		{include file='attachments' objectID=$article->articleID}
+	<div class="entry__content htmlContent" itemprop="description articleBody">
+		{if MODULE_WCF_AD}
+			{unsafe:$__wcf->getAdHandler()->getAds('com.woltlab.wcf.article.inArticle')}
+		{/if}
 		
-		{if !$tags|empty}
-			<ul class="tagList articleTagList">
-				{foreach from=$tags item=tag}
-					<li><a href="{link controller='Tagged' object=$tag}objectType=com.woltlab.wcf.article{/link}" class="tag">{$tag->name}</a></li>
-				{/foreach}
-			</ul>
+		{unsafe:$articleContent->getFormattedContent()}
+		
+		{event name='htmlArticleContent'}
+	</div>
+
+	{event name='afterArticleContent'}
+
+	{include file='entryTags' objectType='com.woltlab.wcf.article'}
+
+	{include file='entryAttachments' objectID=$article->articleID}
+
+	<footer class="entry__footer">
+		{if MODULE_LIKE && ARTICLE_ENABLE_LIKE && $__wcf->session->getPermission('user.like.canViewLike')}
+			<div class="articleLikesSummery">
+				{include file="reactionSummaryList" reactionData=$articleLikeData objectType="com.woltlab.wcf.likeableArticle" objectID=$article->articleID}
+			</div>
 		{/if}
 		
-		<div class="row articleLikeSection">
-			{if MODULE_LIKE && ARTICLE_ENABLE_LIKE && $__wcf->session->getPermission('user.like.canViewLike')}
-				<div class="col-xs-12 col-md-6">
-					<div class="articleLikesSummery">
-						{include file="reactionSummaryList" reactionData=$articleLikeData objectType="com.woltlab.wcf.likeableArticle" objectID=$article->articleID}
-					</div>
-				</div>
-			{/if}
-			
-			<div class="col-xs-12 col-md-6 col-md{if !(MODULE_LIKE && ARTICLE_ENABLE_LIKE && $__wcf->session->getPermission('user.like.canViewLike'))} col-md-offset-6{/if}">
-				<ul class="articleButtons buttonGroup buttonList smallButtons">
+		{hascontent}
+			<ul class="entry__footerButtons buttonGroup buttonList smallButtons" role="menu">
```

Comment:
We’re introducing new classes anyway and change up the layout. This is quite the opportunity to think about this kind of construct, especially with all the historic baggage that `buttonGroup` and `buttonList` come with.

The `<li>` are assigned the role `presentation` which indicates that the markup is not ideal.

I’m not saying that we have to change this but we should at least consider if it makes any sense to swap out the HTML at this point.